WalletWasabi.Fluent/ViewModels/Dialogs/ShuttingDownViewModel.cs

```diff
@@ -1,31 +1,43 @@
+using ReactiveUI;
 using System.Reactive;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
 
 namespace WalletWasabi.Fluent.ViewModels.Dialogs;
 
-[NavigationMetaData(Title = "Wasabi is shutting down...")]
+[NavigationMetaData(Title = "Please wait to shut down...")]
 public partial class ShuttingDownViewModel : DialogViewModelBase<Unit>
 {
 	private readonly ApplicationViewModel _applicationViewModel;
+	private IDisposable? _intervalDisposable;
 
 	public ShuttingDownViewModel(ApplicationViewModel applicationViewModel)
 	{
 		_applicationViewModel = applicationViewModel;
+
+		NextCommand = ReactiveCommand.Create(OnNext);
```

Comment:
If it was a `RoutableViewModel` this could be `NextCommand = CancelCommand`.

Reply:
@soosr fixed in latest commit.